keypom/keypom-airfoil
# chore: internal qa feedback enhancements

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
This PR fixes:
- [x] Disable Coming soon buttons in homepage (show disabled state)
- [x] Move “My drops” to be a full nav item (similar to doc and get in touch) (make sure to check if the nav is responsive in mobile, if its breaks, it should remain in the menu)
- [x] Change “All Drops” to “My Drops”
- [ ] Redirect to “My Drops” after user has connected with the wallet
- [x] use outline variant for secondary cta button (e.g. Master key modal cancel button)
- [x] replace the existing Master key icon in the menu dropdown with a key icon

Fixes #156 

# How to test
Show steps to replicate and test the feature/fixes in this PR
disable coming soon buttons in homepage
1. check the drop section for ticket and nft
2. coming soon should be disabled

my drops should be in nav item
1. Go to the preview link
2. Connect your wallet
3. My Drops should appear as one of the Nav Item

all drops should be called my drops
1. Go to My Drops
2. It should be called My Drops now

outline variant for secondary button
1. try to set master key.
2. cancel button should be outlined.

# Screenshots/Screen Recording of Implementation
[see slack](https://airfoilhq.slack.com/archives/C04DA26740K/p1677575800763329?thread_ts=1677559788.609599&cid=C04DA26740K)
